### DISCUSSION TITLE
Trying to use ocilib for linux to call stored procedures with cursor returns

### DISCUSSION THREAD

**Discussion by sayoojbk** [vrogier/ocilib#304] [Q&A]
I am trying to build the repo from source so I can get the .so required to use ocilib for my C++ project. While building from source. I followed the step 
./configure and this ran successfully
Now i run the Makefile that is created and it shoots up error oledb.h error.(not found error)
PS: any links where i can find prebuilt .so for the same could also be useful.

**Comment by vrogier** [comment]
Have you tried to retrieve the release tar file, decompress it and run the usual:
./configure
./make

Vincent

**Reply by sayoojbk** [reply]
When i do run that it gives up an error that the oracle headers are missing so I have to hardcode the path in the configure file for the same. Same goes for the oci.h header file path. I am actually working on a remote system so cant do much configuration changes to it. But can get the header files and as such when needed.

**Reply by vrogier** [reply]
I don't really get the issue.
Can you provide the configure log ?
